scripts/multiblocktweaker/large_weapons_factory.zs

```diff
@@ -166,16 +167,16 @@ for solder in soldering_alloys {
 	for s_fuel in solidfuels {
 		for s_oxy in solidoxys {
 			for i in 0 to 8 {
-				<multiblock:multiblocktweaker:large_weapons_factory>.recipeMap.recipeBuilder()
+				large_weapons_factory.recipeBuilder()
 					.inputs([<ore:stickAluminium>*4, 
 							<ore:ringAluminium>*2,
 							<ore:plateAluminium>*2,
-							<ore:plateCurvedAluminium>*4,
+							<ore:foilAluminium>*4,
```

Comment:
Note that curved plates are equal to one ingot, and foils are equal to 1/4. Is this cost change intentional.